### DISCUSSION TITLE
Can I see the values of keys being set?

### DISCUSSION THREAD

**Discussion by BharathBK** [jsocol/django-ratelimit#251] [Q&A] [answered]
I am trying to set up a rate-limit to limit the number of times a `POST` request is called.

I have this in my settings:
```
CACHES = {
    "default": {
        "BACKEND": "django_redis.cache.RedisCache",
        "LOCATION": "redis://127.0.0.1:6379/1",
        "OPTIONS": {
            "CLIENT_CLASS": "django_redis.client.DefaultClient",
        },
        "KEY_PREFIX": "my_app"
    }
}
```

I have this in `views.py`

```
@ratelimit(key='ip', rate='1/5m', method=['GET', 'POST'])
def rate_limit_func(request):
     if request.methos == 'POST':
        return Response(status='200')
```

The rate limit is working as expected. But I cannot see any keys getting stored in the redis server
```
127.0.0.1:6379> ping
PONG
127.0.0.1:6379> get ip
(nil)
127.0.0.1:6379> keys *
(empty list or set)
```
I am worried if this will work in production and that I cannot see the keys in the redis server.
Any help is greatly appreciated. Thanks!

**Comment by BharathBK** [comment, marked as answer]
I was looking at the wrong place. By default, redis has databases indexed from 0 to 15. In my Django `settings.py`, I am using database with index 1 for my cache. I can select that database using `redis-cli -n -1`. So,

```
➜ redis-cli -n 1
127.0.0.1:6379[1]> keys *
```

This gave me the list of keys that were being stored by `django-ratelimit`

[This](https://stackoverflow.com/a/13387801/10230224) StackOverflow answer helped me to figure this out.

**Reply by jsocol** [reply]
Great, I’m glad you were able to resolve it. Thank you for passing the solution, and for using the discussion board!
